core/src/main/java/org/elasticsearch/action/admin/cluster/snapshots/get/GetSnapshotsRequest.java

```diff
@@ -112,11 +114,26 @@ public GetSnapshotsRequest snapshots(String[] snapshots) {
         return this;
     }
 
+    /**
+     * Sets if ignore the unavailable snapshots
+     *
+     * @return this request
+     */
+    public GetSnapshotsRequest ignoreUnavailable(boolean ignoreUnavailable) {
+        this.ignoreUnavailable = ignoreUnavailable;
+        return this;
+    }
+
+    public boolean isIgnoreUnavailable() {
```

Comment:
Add Javadoc to the getter. I would also remove the `is***`prefix.

Example Javadoc:

```
/**
 * @return Whether snapshots should be ignored when unavailable (corrupt or temporarily not fetchable)
 */
```